content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -421,14 +421,18 @@ This is useful if you want to target a specific namespace with a label
 
 Type: Annotation
 
-Example: `kubernetes.io/limit-ranger: "LimitRanger plugin set: cpu, memory request for container nginx; cpu, memory limit for container nginx"`
+Example: `kubernetes.io/limit-ranger: LimitRanger plugin set:`                           
```

Comment:
Make sure this is valid YAML.

Reply:
@sftim  it is annotation only and it is valid yaml

Reply:
Hi @kundan2707, how did you verify it?

I tested it locally by creating a limit range, PV, PVC, and pod. The annotation only reflected on the pod but not the PVC.

Kindly provide a single manifest script I can apply and test locally.

Reply:
@kundan2707, new features are usually linked to backend code changes, and updates to the reference doc are made in tandem with these changes. So, adding a new feature to this reference doc alone isn't valid.

I'll suggest you open an issue on the Kubernetes repo suggesting this feature.

Reply:
@network-charles we're always willing to document existing features; see https://kubernetes.io/docs/concepts/policy/limit-range/ which already mentions this behavior.

Reply:
I'd quote the string value here. Otherwise, the document would be invalid.

Reply:
Ok. Yesterday, I simulated the behavior by creating a limit range, PV, PVC, and pod. After requesting the YAML output of the PVC, I didn't see the annotation, but I saw it for the pod. So, it made me think it hasn’t been rolled out yet.